agent/api/container/container.go

```diff
@@ -270,7 +270,7 @@ type Container struct {
 	ContainerArn string `json:"ContainerArn,omitempty"`
 
 	// ExecCommandAgentMetadata holds metadata about the exec agent running inside the container (i.e. SSM Agent).
-	ExecCommandAgentMetadata *ExecCommandAgentMetadata `json:"execCommandAgentMetadata"`
+	ExecCommandAgentMetadata *ExecCommandAgentMetadata `json:"ExecCommandAgentMetadata"`
```

Comment:
this needs a bump in the agent state file version: https://github.com/aws/amazon-ecs-agent/blob/master/agent/statemanager/state_manager.go#L106 (can add a TODO for later too)

Reply:
Done. Thanks for pointing this out.